### ISSUE TITLE
Improve usability of new IIS Step

### ISSUE THREAD

**Issue by michaelnoonan** [OctopusDeploy/Issues#2840]
- [x] Make the meaning of the `Web Site Name` field more clear - in some cases it will create the Web Site if missing, in other cases this must be an existing Web Site.
- [x] Better example of a virtual path (it shouldn't include the Web Site Name... but `/parent1/parent2/etc` makes it look like `parent1` should be the Web Site Name)
- [x] Link to explanation of what virtual path is
- [x] More specific message Virtual Directories / Web Applications explaining what they are (perhaps a link for more information)
- [x] Relative home directory is not really relative, it can be an absolute path so we need a better description there 
- [ ] Customers are confused by Package Id, ideas: provide example, drop down list (can be slow), rename to Package Name, link to library, docs ? Add drop down list similar to accounts?

Source: http://help.octopusdeploy.com/discussions/problems/48103-deployment-error-throw-site-sitename-does-not-exist-in-script-iiswebsite_beforepostdeployps1

Source: http://help.octopusdeploy.com/discussions/problems/47940-problem-creating-new-iis-application-with-new-step-type-deploy-an-iis-web-site

**Comment by Daniel-Svensson** [comment]
Regarding package Id and possible confusion.

The first time I used this new functionality I believed that the IIS deploy step should be added as an additional step after the package step (for example as an extra  child step or similar) in order to facilitate more advanced scenarios such as unpack, modify, take down load balancer , update iis , update loadbalancer scenarios.  However, having to specify the package name and a unique step name quite soon made it evident that this was just another name and icon for the normal package step and we waited until we really needed to make some changes to the deployment process with using the new  step.
